clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/ReadGroupElasticsearchDAOSpec.scala

```diff
@@ -20,7 +20,7 @@ class ReadGroupElasticsearchDAOSpec
   }
 
   it should "updateReadGroupMetadata" in {
-    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3")
+    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3", "GCP")
```

Comment:
@kshakir My philosophy is that test code is the same as any code in a project, and should follow the same rules as any other code. Although strings that only appear once don't necessarily need a constant.

One approach is to replace constant text (or all data) that doesn't need to match with random data. This makes it very clear that the behavior isn't dependent on the data.